Source/carthage/BuildVersion.swift

```diff
@@ -21,7 +21,9 @@ public func localVersion() -> SemanticVersion {
 public func remoteVersion() -> SemanticVersion? {
 	let latestRemoteVersion = Client(.dotCom)
 		.releases(in: Repository(owner: "Carthage", name: "Carthage"), perPage: 1)
```

Comment:
We should probably update this to be 2 per page. Otherwise this will require 2 network requests if the first release is a draft.